src/plugin.js

```diff
@@ -32,6 +32,16 @@ const removeNativePoster = function(player) {
   }
 };
 
+const DESKTOP_SAFARI_VERSION = (function() {
```

Comment:
if/when this gets added to Video.js, we'd probably want to update this to use that first and then fallback to this code.

Reply:
@gkatsev Considering what we discussed yesterday re: the problem with browser version checks returning `null` on the wrong platform, do you think we should keep this as is, or get rid of `return null` here?

Reply:
Also, we are no longer adding DESKTOP_SAFARI_VERSION to video.js right now but maybe it would still be good to be consistent with the other ones? And then making sure to also check `IS_SAFARI`?

Reply:
`IS_SAFARI` will still return true on iOS, so I believe we'd need to add a `IS_DESKTOP_SAFARI` check to avoid the same `null` issue.

Reply:
I guess you could just do `DESKTOP_SAFARI_VERSION && DESKTOP_SAFARI_VERSION >= 11`.

Or if the rest of the team don't mind the loss of consistency update this snippet not to require a null check.

Reply:
Actually, technically the `null` issue only poses a problem if we are checking if the browser version is `less than` a given number since null gets coerced into a 0. Since we're checking `DESKTOP_SAFARI_VERSION >= 11` here maybe we don't need to worry about doing a platform check first?